puz/Grid.cpp

```diff
@@ -135,6 +139,17 @@ Grid::SetupIteration()
             square.m_col = col;
             square.m_row = row;
 
+            if (IsAcrostic() && square.HasNumber()) {

```

Comment:
I haven't completely thought this out yet, but my initial thinking is that this might be better as a separate function since this is called in SetSize and almost every time we aren't going to have squares with numbers yet. Plus it seems like something we would always want to do for acrostics, so we might want to call it in Puzzle::DoLoad instead of in load_jpz.

Reply:
Makes sense - done.